src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/TraceListener.cs

```diff
@@ -176,28 +176,28 @@ public virtual void Close()
             return;
         }
 
-        protected internal virtual string[] GetSupportedAttributes()
+        protected internal virtual string[]? GetSupportedAttributes()
         {
             return null;
         }
 
-        public virtual void TraceTransfer(TraceEventCache eventCache, string source, int id, string message, Guid relatedActivityId)
+        public virtual void TraceTransfer(TraceEventCache? eventCache, string source, int id, string? message, Guid relatedActivityId)
         {
             TraceEvent(eventCache, source, TraceEventType.Transfer, id, message + ", relatedActivityId=" + relatedActivityId.ToString());
         }
 
         /// <devdoc>
         ///    <para>Emits or displays a message for an assertion that always fails.</para>
         /// </devdoc>
-        public virtual void Fail(string message)
+        public virtual void Fail(string? message)
         {
             Fail(message, null);
         }
 
         /// <devdoc>
         ///    <para>Emits or displays messages for an assertion that always fails.</para>
         /// </devdoc>
-        public virtual void Fail(string message, string detailMessage)
+        public virtual void Fail(string? message, string? detailMessage)
```

Comment:
Are you sure that the `message` argument should be nullable?

Reply:
Well at least it allows null, second I took it same as Exception message which is suggested to be nullable